.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
I would argue that this file does not belong to the repo. This is a configuration file of your particular setup with Visual Studio Code. I'd rather put `.vscode/*` in `.gitignore` and remove this file from this PR.

Reply:
OK